dubbo-rpc/dubbo-rpc-dubbo/src/test/java/org/apache/dubbo/rpc/protocol/dubbo/telnet/InvokerTelnetHandlerTest.java

```diff
@@ -137,69 +139,8 @@ public void testComplexParamWithoutSpecifyParamType() throws RemotingException {
         assertTrue(result.contains("result: 12"));
     }
 
-    @SuppressWarnings("unchecked")
-    @Test
-    public void testComplexParamSpecifyParamType() throws RemotingException {
-        mockChannel = mock(Channel.class);
```

Comment:
should we remove UT in refactore? Are those now become invalid or it is something else ?

Reply:
It is invalid